graphql-server/pkg/dataloaders/reaction.go

```diff
@@ -35,11 +47,39 @@ func NewReactionDataloader(reactionQuery queries.IReactionQuery) ReactionDataloa
 		},
 	})
 
+	userProposalReactionLoader := godataloader.NewDataLoader(godataloader.DataLoaderConfig[UserProposalReactionKey, string]{
+		MaxBatch: DefaultMaxBatch,
+		Wait:     DefaultWait,
+		Fetch: func(keys []UserProposalReactionKey) ([]string, []error) {
+			// There should only be once address per proposal
```

Comment:
Assumption should be per request session we serve the same user, so we can ignore rest keys

Reply:
will it be hard for us to handle multiple users here for correctness? otherwise will need to watch out for incorrect usages in the future